### DISCUSSION TITLE
به کجا چنین شتابان

### DISCUSSION THREAD

**Discussion by GoldenVDS** [hiddify/Hiddify-Manager#3144] [Bug باگ: please use issue]
پنل بجای پیش رفت مدام در حال پس رفتن هست.
باگ های متنوعی تا حالا گزارش شده از خطاهای دیتابیس گرفته تا مواردی که کاربران عادی گزارش میکنند. و فقط با وعده در حال بررسی روبرو میشوند.

هر دم از این باغ بری میرسد منتهی خسته تر از خسته ای میرسد!!!.

باگ utls
باگ telegram_bot
اینا حداقل هایی بود که کاربران در نسخ قبلی داشتند و الان ندارند.
به لطف اینهمه سفارش به apply config بخاطر باگ های پنل که موجوده کمک کمک کاربران با عدم دریافت ssl دارند مواجه میشوند.

خطای tls هم سه روزی هست مشهود شده با سرور و کاربر خانگی در خارج کشور هم تست شده مشهود هست.

حیف نسخه 6 که دیباگ شده بود و از دسترس خارج شد.
https://t.me/Goldvds

**Comment by parsatavafi** [comment]
خیلی مردم جالب و حق به جانبی هستیم، بجای تشکر از این همه فعالیت اومدید اون یه ذره ی خالی لیوان رو میبینید و شعر مینویسید، خسته ترین قطعا خود شمایی
مگه شما مدیر این پروژه اید که میفرمایید عقب گرد کنید، شما از طرف خودتون میتونید (نظر) بدید نه این که برای پروژه ی که رایگانه تعیین تکلیف کنید‌. وعده ی در حال پیگیریه میدن؟ خودتون میخونید چی مینویسید یا فقط مینویسید؟ این همه مشکلات بوده و رفع کردند اما شما نمیخواید ببینید
از نسخه دو‌ ماه پیش راضی بودید چرا روی اون نسخه بر نمیگردید؟ مجبور نیستید از اپدیت هایی که شبانه روز زحمت می کشند استفاده کنید که اخر سر بیاید اینجوری بی مهری کنید ...
این همه پنل من نمیدونم اونایی که میام تو تلگرام یا گیت هاب این چیزها رو مینویسند زیر گردنشون چاقویی چیزی گذاشتند؟ استفاده نکن عزیز ... مگه مجبورتون کردند که بعد بیاید نامه ی عاشقانه بنویسید ... اگر هم میخواید استفاده کنید لطفا زحمات دوستان و توسعه دهنده رو لگد مال نکنید، باز هم میگم ما خیلی مردم جالبی هستیم

**Reply by GoldenVDS** [reply]
بله پنل زیاده خیلی هم زیاده از چینی و روسی گرفته تا ایرانی ها ولی هیچکدوم با ارایه آپدیت پس نرفتند بلکه بهتر شدند.

چاقو رو انگار بیخ گلوی شما گذاشتند نه من 

الان کدوم قابلیت پنل بجز هسته xray داره کار میکنه؟ کدوم افزوده داره کار میکنه؟ 
فقظ هی شاخه بی برگ و بر به پنل افزوده شده و همان قابلیت حداقل خودش روهم از دست داده مدام باید سرویس ریستارت شود و یا کانفیگ بازنشانی و پیکربندی شود

کسی طلبی از کسی نداره شما اگه ندیدی و نبودی ما بودیم دیدیم که حتی روی نسخه 4 یک تعداد سودجو این پنل را میامدن لوگو عوض میکردند و میفروختند میدونی چرا؟ چون جذاب بود چون نو اوری داشت چون مخاطب داشت.

کل گله من به شخصه از این بابت که همه این زحمات دو ماهه به بازی گرفته شده اونم توسط خود تیم چون نسخه 6 تقریبا بدون باگ شده بود الان نسخه 7 جمعی که بیس کارشون و روی این پروژه رفتند جلو گرفتار کرده.

شما فکر میکنی همه vpn فروشند؟ خیر اقا هستند کسانیکه سرور خارج کشور تهیه کردن و رایگان در اختیار برخی گذاشتند که بتوانند آنها نیز اینترنت آزاد  رو به دیگران هدیه دهند چون هدف اینجور پروژه ها کمک به دیگران هست و برخی هم دست به کاربردند تا این کمک بیشتر نشر پیدا کنه.

**Comment by parsatavafi** [comment]
عزیز جان مشکلت رو در جای دیگه ای دنبالش بگرد، تمام قابلیت های پنل برای بنده اکتیوه و بیشتر از 600 تا مشترک راضی روش هست و هیچ کدوم هم گله ای ندارند. تنها مورد این چند  وقت قطعی شبانه پنل روی نسخه 7.1.3 بود که با اپدیت 7.1.4 رفع شد
شما از کجا میدونی من نبودم و ندیدم؟! شما اگه نسخه 4 دیدید من روی نسخه 3 بالای دویست مشتری داشتم... اون موقع که توسعه دهنده دنبال تکمیل پنل بود داشتم سعی می کردم بهشون کمک کنم برای ساخت ویدئوهای اموزشی (که البته خیلی هم موفق نبودم و باید حق رو گفت)
شما مشکلت از جای دیگه ای اب میخوره ...
این که کسی اومده کانفیگ ران کرده که به مردم کمک کنه (ما که اندازه ی انگشت های یک دست هم ندیدیم ولی باشه شما درست میگید، ارتباطش با شما چیه وقتی خودت اومدی ادرس تلگرام دادی زیر نوشتت و فروش سرور و VPS و VDS داری!) نیازی به هیدیفای نداره، یه سر به توئیتر سگارو بزنید اسکریپت هایی رو معرفی کرده که کمتر از چند دقیقه کانفیگ میشند. رفیق باز هم تاکید می کنم مشکلت از جای دیگه ای هست ... خودت میدونی و بچه های هیدیفای ولی این مشکلاتی که شما گفتید رو با رول بک به نسخه های قبلی میتونید حلش کنید به گفته ی خودتون دیگه حتما ... پس حق ندارید گله کنید که چرا اپدیت می کنید و شاخ و برگ زیاد می کنید و براشون شعر بنویسید و این شکلی طلبکارانه با کسی که داره لطف می کنه در حق اینترنت کشور صحبت کنید. باز هم خودتون میدونید و بچه های هیدیفای ولی من به عنوان یه ادمی که هم اون بچه ها رو از روز اول کارشون رو دیدم، دستشون رو میبوسم و اصلا مثل شما طلبی ازشون ندارم ... حالا شما فکر کن چاقو زیر گلومه که اینا رو براشون بنویسم ... :))) reply نزنید لطفا هرچی شما میگید درسته و کاملا حق با شماست ...

**Reply by GoldenVDS** [reply]
پس سرور شما هسته سینگ باکسش درست کار میکنه و پروکسی تلگرام کار میکنه و محاسبه حجم داره Shadowsocks و ShadowTLS هم کار میکنه محاسبه حجم داره حتی v2ray هم فعاله کار میکنه و محاسبه حجم هم میکنه.
عجب پس درخت های باغ ما شاخه بی برگ و بر داره باغ شما آباده.
درضمن من مشکلی ندارم اتفاقا شب تا صبح و صبح تا شب دارم مشکلات بقیه رو حل میکنم دونیت و پول و اجرتم نمیگیرم.

و متاسفم که مفروض کردی طلبکارانه بر خورد میکنم.

همینقدر از جهل خودت بگم اولا مشکل قطعی نیمه شب با اپدیت 7.1.3 حل شد حل که صورت مسیله پاک شد کرونجاب مربوط به رینیو ssl حذف شد اونم پاک کردن صورت مسیله است نه حل اون چون اگر پنل باگ نداشته باشه و روزی ده هابار ریست و باز چینی نیاز نداشته باشه اون کرون ضروری هست برای اعتبار به ssl و تمدید اون. پس چیزی حل نشده فقط مسئله پاک شده.
فعلا صدقه سری روزی چندبار نصب و بازچینی بهش نیازی نیست.
اپدیت 7.1.4 هم رفع دسته گلی بود که خود تیم میدونه چیه لازم نیست بیایم جار بزنیم.
میدونی این پنل چقدر وابستگی گیت هابش داره؟ میدونی فردا روزی این اکانت بسته بشه یا حذف بشه حال تو یکی دیدن داره با این همه اهن و تلم که اینقدر مشتری دارم.:))
نحوه محاسبه حجم کاربر رو دیدی و درکی ازش داری؟ دیدی چه پروسه سنگینی داره؟ دیدی کرون جاب هر دقیقه برای اجرای اون چه فشاری به سرور میاره. البته بجای حل اون هم توصیه میکنند برید ارتقا دهید.

در اخر شما ریپلای زدی پاسخ دادم باز ریپلای زدی بازهم پاسخ دادم پس حق منع کردن نداری حالا از ائنکه چاقو زیر گلوت گذاشته بپرس من درست میگم یا نه.


اینا دست نجونبانند این پروژه خاطره انگیز و ستودنی به تار عنکبود مینشینه و شروع شده درباره این پروژه بد گویی و علامت و سوال و غیره.

شما همون سایر پنل ها رو ببین وقتی اعلام باگ میشه صاحب اثر اعلام تشکر میکنه و میره رفع میکنه و میاد میگه به لطف گزارش فلانی به همت فلان مورد و بی چشم داشت میشینه کد میزنه اصلاح میکنه

اینجا چی جواب ناراحتی برو پنل دیگه نمیدونم ما مجانی داریم کار میکنیم اینا همه درسته ولی اینکارها رو کردید هدفی داشتید امیدوارم به مقصد برسید بار کج به منزل نمیرسه.

در خصوص ادرس تلگرام هم گذاشتم تا اونیکه در جهل کامل داره راجب مسایل شبکه و سرور به ملت اطلاع میده و دستور میده و هی میگه من ادمینم سه ماهه دارم فلان میکنم و بر خورد و جواب ناثواب توی گروه میده بدون من جای دیگه میتونم حرفم و بزنم.

درضمن من ایران نیستم و vps , vds هم برای ایران ندارم و اصلا فروشنده نیستم بر عکس من سر تیم ساپورت یکی از دیتاسنترها هستم صدقه سری ایرانیانی که سرور مگیرن یا با گوگل ترنسلیت میان متن میزارند شبیه دعوت به عروسی بجای تیکت با فینگلیش متن میزارند اوناهم دیدن بالغ بر هزار مشتری ایرانی دارند پشتبان فارسی زبان استخدام کردن.

بگذریم با امید توفبق روز افزون

**Comment by moboneir** [comment]
سلام واقعا نسخه آخر هیدیفای منو روانی کرده

باگ ssl که ساب بروز نمیشه و پچ نمیشه
باگ اعمال تغییرات 
باگ ریستارت و بروزرسانی تو پنل
باگ cdn که وصل نمیشه و eof میده

اقا نسخه ۱۲ ک مشکل نداشت چرا اینهمه باگ زدین و اپدیت کردین اصلا اپدیت نمیدن ک باگ برطرف بشه

با تشکر
